packages/veritone-widgets/src/build-entry.js

```diff
@@ -7,23 +7,20 @@ export OAuthLoginButton, {
   OAuthLoginButtonWidget
 } from './widgets/OAuthLoginButton';
 export FilePicker, { FilePickerWidget } from './widgets/FilePicker';
+export filePickerReducer, * as filePickerModule from './redux/modules/filePicker';
+export filePickerSaga from './redux/modules/filePicker/filePickerSaga';
 export { EngineSelectionWidget } from './widgets/EngineSelection';
-export { TableWidget } from './widgets/Table';
+export { TableWidget as Table, TableWidget  } from './widgets/Table';
```

Comment:
Table would be a plain component, not a wrapped widget -- so this change doesnt make sense

Reply:
agreed. i'll remove it